### DISCUSSION TITLE
Subscription System

### DISCUSSION THREAD

**Discussion by ghost** [thuliteio/doks#737] [Ideas]
Hi, 
Subscription system is appreciated by the blogger, digital creators like Ghost cms.
How we can integrate a subscription system (stripe, buy me a coffe) inside this theme ?
Any solutions to share ?
Thanks

**Comment by h-enk** [comment]
Definitely interesting. Can someone recommend a solid approach /solution?

**Comment by constantcopyandwork** [comment]
I have been using Stripe payment links, only. Nothing fancy, just a button and selector/dropdown. But Stripe does allow for subscription. 
And they have this page for more info on [Stripe API](https://stripe.com/docs/payments/payment-links/api).
I'd definitely like this.

**Comment by h-enk** [comment]
Yes, I also like Stripe — I would surely go down that road. 

I stumbled upon an interesting blog post by Jason Lengstorf, called [Manage Subscriptions and Protect Content With Stripe](https://www.netlify.com/blog/2020/07/13/manage-subscriptions-and-protect-content-with-stripe/) (July 13, 2020) — note that you would need a  Netlify Business plan ($99 per member/month) for role-based access control though. So, I'm still looking for a solid, cheaper way.

**Reply by constantcopyandwork** [reply]
I found this post too: https://discourse.gohugo.io/t/easy-full-hugo-simple-online-shop-with-just-netlify-stripe/27960
Might be interesting to take a look at their approach. I don't think I understand enough of Netlifly to tell if it's doable.

**Reply by h-enk** [reply]
A product store with physical or digital goods is easy to set up, thanks to Stripe + Netlify functions. Managing subscriptions is also easy thanks to Stripe. The challenge here is in managing gated content in a solid, easy way  — think a database + API's + functions.
